### PR TITLE
Sympy-10313: Updated printing.rst

### PR DESCRIPTION
**Pull request by abhishekgarg16**
Updated printing.rst, added an example for dot printing.
Fix for [#10313](https://github.com/sympy/sympy/issues/10313)

### CODE REVIEW

**Comment by abhishekgarg16** [comment]
@shivamvats , can you please look into the pull request and let me know why these tests are failing?

File: doc/src/tutorial/printing.rst [hunk 1]
```diff
@@ -229,6 +229,32 @@ format, which can be rendered with Graphviz.  See the
 :ref:`tutorial-manipulation` section for some examples of the output of this
 printer.
 
+    >>> from sympy.printing.dot import dotprint
+    >>> from sympy.abc import x
+    >>> print(dotprint(x+2)) 
+    digraph{
+
```
Comment (sampadsaha5) [inline doc/src/tutorial/printing.rst]: replace L236, L240, L244, L248, L252 with `<BLANKLINE>`.

I think this would let your test cases pass.

**Comment by sahilshekhawat** [comment]
Hi @abhishekgarg16  ,
Thank you for the PR.

For failing builds, you can always look at the travis errors logs. Also, It is recommended that you run tests locally before pushing. Please refer here: https://github.com/sympy/sympy/wiki/Running-tests

I think the error is due to the reason that an empty line marks as the end of output and therefore, it is treating line after ``digraph{`` as the output of the command, which is incorrect. You need to take care of blanklines. For reference: https://wiki.python.org/moin/MultiLineStringsInDocTest You can update it with:
```
    digraph{
    <BLANKLINE>
    # Graph style
    "ordering"="out"
    "rankdir"="TD"
    <BLANKLINE>
    #########
    # Nodes #
    #########
    <BLANKLINE>
    "Add(Integer(2), Symbol(x))_()" ["color"="black", "label"="Add", "shape"="ellipse"];
    "Integer(2)_(0,)" ["color"="black", "label"="2", "shape"="ellipse"];
    "Symbol(x)_(1,)" ["color"="black", "label"="x", "shape"="ellipse"];
    <BLANKLINE>
    #########
    # Edges #
    #########
    <BLANKLINE>
    "Add(Integer(2), Symbol(x))_()" -> "Integer(2)_(0,)";
    "Add(Integer(2), Symbol(x))_()" -> "Symbol(x)_(1,)";
    }
```

**Comment by moorepants** [comment]
This looks fine.

**Pull request merged**